motion.gemspec

```diff
@@ -25,8 +25,9 @@ Gem::Specification.new do |spec|
   spec.files = Dir["lib/**/*"]
   spec.require_paths = ["lib"]
 
-  # Required for the squiggly HEREDOC
-  spec.required_ruby_version = Gem::Requirement.new(">= 2.3.0")
+  # The lowest version of Ruby against which Motion is tested. See `.travis.yml`
+  # for the full matrix.
+  spec.required_ruby_version = Gem::Requirement.new(">= 2.5.8")
```

Comment:
If we're supporting Ruby 2.5, WDYT if we allow any version of 2.5?

Reply:
Fair enough, I put 2.5.8 here because that is what the tests are running against, but patch-level fixes are probably fine.